src/web/src/components/utilBar/userButton/UserButton.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import { Drawer, Box, Typography } from '@mui/material';
+import { IconButton } from '@mui/material';
+import PersonOutlineIcon from '@mui/icons-material/PersonOutline';
+
+const UserButton = () => {
+  const [showMembers, setShowMembers] = useState(false);
+  return (
+    <>
+      <IconButton
+        type="button"
+        onClick={() => {
+          setShowMembers(true);
+        }}
+      >
+        <PersonOutlineIcon></PersonOutlineIcon>
```

Comment:
```suggestion
        <PersonOutlineIcon />
```
Icon should be self-closing